migrations/v7.js

```diff
@@ -0,0 +1,109 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-accordion - v5.3.0 > v7.0.0', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v5.3.0', { name: 'adapt-contrib-accordion', version: '<=7.3.0' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = content.filter(({ _component }) => _component === 'accordion');
+    if (accordions) return true;
+  });
+
+  /**
+   * * Add JSON field to component and set attribute.
+   */
+  mutateContent('adapt-contrib-accordion - add accordion._items._imageAlignment', async () => {
+    accordions.forEach(accordion => {
+      accordion._items.forEach(item => {
+        item._imageAlignment = 'full';
+      });
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check accordion._items._imageAlignment atrribute', async () => {
+    const isValid = accordions.every(accordion =>
+      accordion._items.every(item =>
+        item && item._imageAlignment === 'full'
+      )
+    );
+    if (!isValid) throw new Error('adapt-contrib-accordion - _imageAlignment not added to every instance of accordion._items');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-accordion - update to v7.3.0', { name: 'adapt-contrib-accordion', version: '7.3.0', framework: '>=5' });
+});
+
+describe('adapt-contrib-accordion - v7.3.0 > v7.7.0', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v7.3.0', { name: 'adapt-contrib-accordion', version: '<=7.7.0' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = content.filter(({ _component }) => _component === 'accordion');
+    if (accordions) return true;
+  });
+
+  /**
+   * * Add field to each item in a JSON array and set attribute.
+   */
+  mutateContent('adapt-contrib-accordion - add accordion._items._imageAlignment', async () => {
```

Comment:
This was done in the previous migration?

Reply:
Yes. I have deleted the duplicate.